frontend/packages/dev-console/src/components/charts/HorizontalStackedBars.scss

```diff
@@ -1,12 +1,22 @@
+$gapSize: 3px;
+
 .odc-horizontal-stacked-bars {
-  display: inline-flex;
-  flex-direction: row;
+  display: inline-block;
   height: 100%;
-  outline: none;
+  overflow: hidden;
   vertical-align: middle;
   width: 100%;
 
+  &__bars {
+    display: flex;
+    flex-direction: row;
+    height: 100%;
+    outline: none;
+    width: calc(100% + #{$gapSize});
+  }
+
   &__data-bar {
+    box-shadow: inset -#{$gapSize} 0 0 #ffffff;
```

Comment:
```suggestion
    box-shadow: inset -#{$gapSize} 0 0 var(--pf-chart-global--Fill--Color--white);
```

Reply:
I had the same comment to @christianvogt - but I was told a direct white was fine.